packages/component-library/src/components/plmg-tooltip/plmg-tooltip.types.ts

```diff
@@ -0,0 +1,17 @@
+export const backgroundColors = ['neutral', 'primary'] as const;
+export type backgroundColors = typeof backgroundColors[number];
```

Comment:
Use unique names for these two declarations.
```suggestion
export const backgroundColors = ['neutral', 'primary'] as const;
export type PlmgTooltipBackgroundColor = typeof backgroundColors[number];
```